src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,24 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence using the Fisher-Yates shuffle algorithm.
```

Comment:
In my opinion, if you state in a piece of documentation that a function has a certain quality, then that's a guarantee you make. It would be a breaking change to subsequently change this. 

That's my position on the matter, but @kurtschelfthout may be of a different opinion.